aten/src/ATen/core/dispatch/Dispatcher.cpp

```diff
@@ -102,6 +102,33 @@ RegistrationHandleRAII Dispatcher::registerDef(FunctionSchema schema) {
         op.operatorIterator_->op.registerSchema(std::move(schema));
       } else {
         TORCH_CHECK(op.schema() == schema, "Tried to register multiple operators with the same name and the same overload name but different schemas: ", schema, " vs ", op.schema());
+        // If the *new* schema is the default alias analysis kind, for BC, we
```

Comment:
Dispatcher::registerDef is becoming very long, can we split this into named subfunctions for readability?

Reply:
This feels a little unfair: most of the lines are devoted to making good error messages :) Do you have an alternative structure in mind?